tests/WE2E/test_configs/verification/config.MET_ensemble_verification_only_vx_time_lag.yaml

```diff
@@ -32,9 +29,9 @@ global:
   ENS_TIME_LAG_HRS: '[ 0, 12 ]'
 
 platform:
-  CCPA_OBS_DIR: '/scratch2/BMC/fv3lam/ens_design_RRFS/obs_data/ccpa/proc'
-  MRMS_OBS_DIR: '/scratch2/BMC/fv3lam/ens_design_RRFS/obs_data/mrms/proc'
-  NDAS_OBS_DIR: '/scratch2/BMC/fv3lam/ens_design_RRFS/obs_data/ndas/proc'
+  CCPA_OBS_DIR: '{{ workflow.EXPTDIR }}/obs_data/ccpa/proc'
```

Comment:
Can we make these the defaults in `config_defaults.yaml`?  Seems like this is the logical default place obs should be placed in.

Reply:
@gsketefian I have made this the default, and added some special functionality to run_WE2E_tests.py to overwrite with the machine file values in the test specifically. However, this still has to be specified here for supported platforms because otherwise the Machine File value would overwrite the default (users will *not* have to specify these variables, this is only for the testing script). This is kind of a chicken-egg problem if we want to both have staged data and the ability to automatically retrieve custom data.